src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -61,6 +65,9 @@ protected ReadThreadingAssemblerArgumentCollection getReadThreadingAssemblerArgu
         return new HaplotypeCallerReadThreadingAssemblerArgumentCollection();
     }
 
+    @Argument(fullName = PLOIDY_REGIONS_NAME, shortName = PLOIDY_REGIONS_NAME, doc = "Interval file with column specifying desired ploidy for genotyping models.", optional = true)
```

Comment:
include in the description here some more detail about how this interacts with the other --ployidy argument

Reply:
Added more to the description